### ISSUE TITLE
Make the "saved" notification disappear on edit

### ISSUE THREAD

**Issue by draganescu** [WordPress/gutenberg#15132]
<img width="1006" alt="Screenshot 2019-04-23 at 23 00 20" src="https://user-images.githubusercontent.com/107534/56611907-aaf76180-661b-11e9-8f00-275a2e66ee30.png">

**Is your feature request related to a problem? Please describe.**
There is an oldish [Trac ticket](https://core.trac.wordpress.org/ticket/32228) about this but I think it doesn't apply anymore code-wise, but the behaviour still exists in Gutenberg.

Briefly when a piece of content is saved, we get this green bar that says: "Post updated.". The bar stays there forever unless you close it with the mouse.

**Describe the solution you'd like**
The bar should disappear after a while or on first edit.

**Comment by karmatosed** [comment]
I am removing the 'User Experience (UX)' label as part of a label cleanup. It's not being used anymore consistently so let's try and keep to 'needs design' and 'needs design feedback'. If we find a need for another label we can consider it but having those 2 should cover this.

**Comment by afercia** [comment]
Discussed during today's accessibility bug scrub. Not sure if this is still relevant, as we've heard there are ongoing explorations with the "snackbars".

In any case, there are two concerns:
- Content (in this case important notices) that disappears on a time based interval [violates a specific WCAG guideline](https://www.w3.org/TR/WCAG21/#timing-adjustable). Rationale: some users may need more time to get to the content. Intervals should be adjustable.
- Potential focus loss: if the "View Post" link is focused and the notice gets removed from the DOM, there's a focus loss. This should be avoided at all costs.

**Comment by draganescu** [comment]
This will be solved by #15594 by having any notification disappear in 10s. It is not exactly as described here but it remains to be seen if this behaviour will be needed.

**Comment by afercia** [comment]
Thanks for pointing it out. Worth noting the accessibility of the so called "snackbars" is to evaluate yet, as there are serious concerns there. (see the non-adjustable timeout to start with).
